input/OFCB/1/Batched/1.xml

```diff
@@ -20,7 +20,7 @@
             <Source>
                 <outcommod>UOX</outcommod>
                 <outrecipe>UOX</outrecipe>
-                <throughput>1000</throughput>
+                <throughput>1e100</throughput>
```

Comment:
Ok! I have more typically seen this as closer to the max double value of 1e300.  But this is fine too.